pkgs/applications/version-management/git-and-tools/pass-git-helper/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, buildPythonPackage, isPy3k, fetchFromGitHub, pyxdg }:
+
+buildPythonPackage rec {
```

Comment:
okay. Now we have to decide if it is a python package that you want to use a dependency or an application where we use the executable from. This seems to be the latter one. In this case use `buildPythonApplication` and reference it in `pkgs/applications/version-management/git-and-tools/default.nix`. It will be then exposed as `gitAndTools.pass-git-helper`.

Reply:
yes.